cdp_scrapers/instances/seattle.py

```diff
@@ -84,6 +87,16 @@ def __init__(self):
             person_aliases=PERSON_ALIASES,
         )
 
+        try:
+            urlopen("https://seattlechannel.org/")
+        except URLError:
+            pass
+        else:
+            raise Exception(
+                "seattlechannel.org may have fixed their SSL cert. "
+                "Check and fix 'requests.get(*, verify=False)' calls"
```

Comment:
Welcoming any other better way to be notified when we can go back to proper https.